lexer.py

```
@@ -0,0 +1,100 @@
+@dataclass
+class Token:
+    type: TokenType
+    value: str
+    position: int
+
+
+end = chr(0)
+
+class Lexer:
+
+    def __init__( self, filename: str ) -> None:
```

Flake8 (7.2.0)

[error] whitespace after '('

(E201)